sql/core/src/main/java/org/apache/spark/sql/sources/v2/reader/ReadSupport.java

```diff
@@ -45,9 +45,6 @@
    * Note that, this may not be a full scan if the data source supports optimization like filter
    * push-down. Implementations should check the input {@link ScanConfig} and adjust the resulting
    * {@link InputPartition input partitions}.
-   *
-   * If this method fails (by throwing an exception), the action will fail and no Spark job will be
-   * submitted.
```

Comment:
Why remove this here? Isn't it an accurate statement?

Reply:
to address https://github.com/apache/spark/pull/22009#discussion_r208983568 , I moved this statement to the classdoc instead of each method.